test/test_unit_user_logic.py

```diff
@@ -84,6 +98,7 @@ def test_rotate_other_api_key_returns_new_key(self):
     '''
     Validates that a new personal API key can be returned
     '''
+
```

Comment:
remove these lines between comments and test

Reply:
Removed. Seems my pycharm linter somehow does it. Do you know how to disable it?